core/modules/views/includes/view.inc

```diff
@@ -310,12 +312,12 @@ class view {
   /**
    * Perform automatic updates when loading or importing a view.
    *
-   * Over time, some things about Views or Backdrop data has changed.
-   * this attempts to do some automatic updates that must happen
-   * to ensure older views will at least try to work.
+   * Over time, some things about Views or Backdrop data has changed. This
+   * attempts to do some automatic updates, that must happen to ensure older
```

Comment:
No comma needed here.

Reply:
(line 316)

Reply:
W.o any punctuation that sentence is big, and makes no sense with 4 verbs in it - some punctuation is needed I think. Perhaps this?:

> This attempts to do some automatic updates that must happen, to ensure older views will at least try to work.

Reply:
Yes, that's fine.